src/Rings/MPolyQuo.jl

```diff
@@ -1046,11 +1046,11 @@ function divides(a::MPolyQuoRingElem, b::MPolyQuoRingElem)
   Q = parent(a)
   J = oscar_groebner_basis(Q)
 
-  BS = IdealGens([a.f], keep_ordering = false)
```

Comment:
The constructor for `MPolyQuo` has a monomial ordering as a third argument. Here I came to appreciate it: In my application I use a block ordering with first block the "Rabinowitsch-variables" and then the original ones. 
It proved to be essential to use this ordering in the computations; for `degrevlex` everything died. 

However, it seems that not all internal methods use the ordering provided in the constructor. Some examples came through after changing these arguments here to `true`, so I assume it had the anticipated effect that my block ordering was used instead. Could you confirm that this is what it is supposed to do and that my changes don't do any harm here?

Reply:
I quote from the type definition of MPolyQuoRing in MPolyQuo.jl (lines 13-14):
``` 
  #= the current design decision is to fix the ordering to be default_ordering(R) =#
  #= that is, the user is not allowed to change this with the outer constructor quo =#
```
So your impression is correct. There are design decisions denying the user influence on the ordering.